### DISCUSSION TITLE
[Support]: Standard Docker compose installation for Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by csvke** [blakeblackshear/frigate#13282] [General Support] [answered]
### Describe the problem you are having

Can't fetch with no manifest for linux/arm/v8
`uname -a` returns
`Linux rpi4 6.6.31+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.31-1+rpt1 (2024-05-29) aarch64 GNU/Linux`

### Version

0.14.0

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
no matching manifest for linux/arm/v8 in the manifest list entries
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment]
We need to see the docker compose

**Reply by csvke** [reply]
It's just the standard docker-compose.yaml suggested by the documentation.

**Reply by NickM-27** [reply, marked as answer]
The standard compose has fields that you need to fill in for yourself. If you don't want to provide the information requested then we can't help

**Reply by csvke** [reply]
I think it's because I tried to install Frigate in an exFAT volume (I maybe wrong) - but I have moved the project back to an Radxa Rock5b now. Thanks for the help. Please close the issue.
